excel/beginner/basics-of-excel/cells-continued.md

```diff
@@ -0,0 +1,78 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# Cells (Find and Replace)
+
+---
+## Content
+
+Cells are essentially where all of your data goes. So finding and replacing mistakes should be easy.

```

Comment:
Refrain from using assumptive language such as  "this is easy".

In this particular insight, I think we can just remove these two sentences and have the insight start with the one below about Google Docs and Excel

Reply:
This says. Finding and replacing mistakes should be easy..
It states that it "Should" be easy to find cells and data within cells. So, the sole purpose of this function is to "make it easy" to find and replace/

Nothing in the lecture suggests something is easy. However, the sole purpose of the find and replace is to make it easier to find and manipulate data in cells